docs/deployment-ux.md

```diff
@@ -300,111 +310,193 @@ impacts
     running everywhere:
 
-    ``` {.shell}
-    $ aws ecs describe-environment-status --environment-name SomeEnvironment/Prod --table
-    REVISION                  STATUS       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
-    SomeEnvironment/Prod:1    Undeploying  0         5         5            5        
-    SomeEnvironment/Prod:2    Deploying    5         1         1            1        
+    ```
+    $ aws ecs describe-environment-status --all --environment-name SomeEnvironment/Prod --table
+    REVISION                  STATUS       DESIRED   CURRENT
+    SomeEnvironment/Prod:1    Undeploying  0         5
+    SomeEnvironment/Prod:2    Deploying    5         1
     ```
 
     First task in old revision is terminated:
 
-    ``` {.shell}
-    $ aws ecs describe-environment-status --environment-name SomeEnvironment/Prod --table
-    REVISION                  STATUS       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
-    SomeEnvironment/Prod:1    Undeploying  0         4         4            4        
-    SomeEnvironment/Prod:2    Deploying    5         1         1            1        
+    ```
+    $ aws ecs describe-environment-status --all --environment-name SomeEnvironment/Prod --table
+    REVISION                  STATUS       DESIRED   CURRENT
+    SomeEnvironment/Prod:1    Undeploying  0         4
+    SomeEnvironment/Prod:2    Deploying    5         1
     ```
 
     Deployment progresses in a similar fashion until the last task of
     the old revision is left:
 
-    ``` {.shell}
-    $ aws ecs describe-environment-status --environment-name SomeEnvironment/Prod --table
-    REVISION                  STATUS       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
-    SomeEnvironment/Prod:1    Undeploying  0         1         1            1        
-    SomeEnvironment/Prod:2    Deploying    5         5         5            5        
+    ```
+    $ aws ecs describe-environment-status --all --environment-name SomeEnvironment/Prod --table
+    REVISION                  STATUS       DESIRED   CURRENT
+    SomeEnvironment/Prod:1    Undeploying  0         1
+    SomeEnvironment/Prod:2    Deploying    5         5
     ```
 
     Deployment completes once the final task in the old revision is
     terminated:
 
-    ``` {.shell}
-    $ aws ecs describe-environment-status --environment-name SomeEnvironment/Prod --table
-    REVISION                  STATUS       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
-    SomeEnvironment/Prod:1    Inactive     0         0         0            0        
-    SomeEnvironment/Prod:2    Active       5         5         5            5        
     ```
+    $ aws ecs describe-environment-status --all --environment-name SomeEnvironment/Prod --table
+    REVISION                  STATUS       DESIRED   CURRENT
+    SomeEnvironment/Prod:1    Inactive     0         0
+    SomeEnvironment/Prod:2    Active       5         5
+    ```
+
+#### Changes to cluster membership while a deployment is ongoing
```

Comment:
I think we should break this out into sections: how deployment behaves when there are no cluster changes and how do we handle instances being added/removed and task failures.

Reply:
I think the "deployment strategies" section basically illustrates the "what happens when the cluster doesn't change"? I can rename that to reflect that it's the happy path, and then break this section out into separate scenarios for both deployment strategies. Come to think of it, the behaviour might be different, depending on strategy, so that's a good move either way.

Would that cover it?